pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/io/WritableSequentialData.java

```diff
@@ -293,14 +327,8 @@ default void writeLong(final long value) throws BufferOverflowException, Uncheck
         if (remaining() < Long.BYTES) {
             throw new BufferOverflowException();
         }
-        writeByte((byte)(value >>> 56));
-        writeByte((byte)(value >>> 48));
-        writeByte((byte)(value >>> 40));
-        writeByte((byte)(value >>> 32));
-        writeByte((byte)(value >>> 24));
-        writeByte((byte)(value >>> 16));
-        writeByte((byte)(value >>>  8));
-        writeByte((byte)(value));
+        writeByte4((byte) (value >>> 56), (byte) (value >>> 48), (byte) (value >>> 40), (byte) (value >>> 32));
+        writeByte4((byte) (value >>> 24), (byte) (value >>> 16), (byte) (value >>> 8), (byte) (value));
```

Comment:
Minor: might as well add `writeByte8`.

Reply:
It can be done, but looks like too corner case to me :)